config/config.go

```diff
@@ -1612,115 +1622,129 @@ func (c *Config) resetMissingTomlFieldTracker() {
 	c.toml.MissingField = c.missingTomlField
 }
 
-func (c *Config) getFieldString(tbl *ast.Table, fieldName string, target *string) {
+func (c *Config) getFieldString(tbl *ast.Table, fieldName string) string {
 	if node, ok := tbl.Fields[fieldName]; ok {
 		if kv, ok := node.(*ast.KeyValue); ok {
 			if str, ok := kv.Value.(*ast.String); ok {
-				*target = str.Value
+				return str.Value
 			}
 		}
 	}
+
+	return ""
 }
 
-func (c *Config) getFieldDuration(tbl *ast.Table, fieldName string, target interface{}) {
+func (c *Config) getFieldDuration(tbl *ast.Table, fieldName string) (bool, time.Duration) {
```

Comment:
Can we please adhere to the Golang "convention" of ordering the return value as `(value, found)`?

Reply:
Sure.